src/utils/hibp.js

```diff
@@ -247,16 +233,35 @@ async function subscribeHash (sha1) {
   return await kAnonReq(path, options)
 }
 
+/**
+A range subscription can be deleted with the following request:
+DELETE /range/[hash prefix]
+
+There is one possible response code that can be returned:
+1. HTTP 200: Range subscription already exists
+
```

Comment:
This is valid JS but I think for JSDoc you want something more like:

```suggestion
/**
 * A range subscription can be deleted with the following request:
 * DELETE /range/[hash prefix]
 *
 * There is one possible response code that can be returned:
 * 1. HTTP 200: Range subscription already exists
 *
```

Reply:
Fixed